markdown/dev/reference/api/part/en.md

```diff
@@ -7,6 +7,27 @@ Parts truly are the building blocks of FreeSewing as they not only provide
 the configuration, but also a `draft()` method that does the actual work
 of drafting a parametric design.
 
+## Properties
+
+A Part object comes with the following properties:
+
+- `attributes` : An [Attributes](/reference/api/attributes) instance holding
+  the part's attributes
+- `hidden` : When this is `true` the part will be hidden (excluded it from the
```

Comment:
I've spaced on how to alter the text here but I believe this should read 'excluding' rather than 'excluded'

Reply:
Feel free to submit a PR to fix that.